docs/linux_dependencies.md

```diff
@@ -55,6 +55,22 @@ Add your arch to the end of the package to remove the linker error. For example:
 sudo dnf install alsa-lib-devel.x86_64
 ```
 
+## Fedora / Silverblue
+
+```bash
+toolbox enter
+sudo dnf install gcc-c++ libX11-devel alsa-lib-devel systemd-devel wayland-devel libxkbcommon-devel
```

Comment:
not a linux or fedora expert, but it feels strange to install both `libX11-devel` and `wayland-devel`

Reply:
Strange, winit enables its `wayland` and `x11` features by default, presumably using x11 as a fallback if wayland is not available, but Bevy only enables winit's `x11` feature by default.
Not sure why?

I do think it makes sense to have both `libX11-devel` and `wayland-devel`, otherwise if someone decides to later enable Bevy's `wayland` feature they'll by surprised by this missing dependency.